README.md

```diff
@@ -73,9 +73,16 @@ Please feel free to checkout [Canvasbird Contribution Guidelines](https://github
 
 Thanks goes to these wonderful people ✨✨:
 
+<a href="https://github.com/Canvasbird/canvasboard/graphs/contributors">
```

Comment:
Clicking on an image will take us to the `canvasboard` contribution graph. Instead can we please direct it to the particular user's profile?